research/conditional/train/cc_train.py

```diff
@@ -220,6 +226,8 @@ def main(
         min_eval_group_size=args.min_eval_group_size,
         max_eval_group_size=args.max_eval_group_size,
         steps_until_start_temperature_learn=args.steps_until_start_temperature_learn,
+        zipped_model_fits_params=zipped_model_fits_params,
```

Comment:
How about we unpack this to a list already? so the arguments are as self-explanatory as possible.

Reply:
unpack before***